### PR TITLE
[#157485217] add fiscalcode to service

### PR DESCRIPTION
**Pull request by gunzip**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#157485217](https://www.pivotaltracker.com/story/show/157485217): Come SAS voglio poter comunicare il mio codice fiscale, in modo da poterlo utilizzare durante i pagamenti a mio favore

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api/definitions.yaml [hunk 1]
```diff
@@ -368,13 +368,21 @@ Service:
     department_name:
       $ref: "#/DepartmentName"
     authorized_cidrs:
+      description: |-
+        List of IPs (or IP ranges) that are authorized
+        to use the APIs on behalf of this service account
```
Comment (cloudify) [inline api/definitions.yaml]: Allowed source IPs or CIDRs for this service.

File: api/definitions.yaml [hunk 2]
```diff
@@ -368,13 +368,21 @@ Service:
...
       type: array
       items:
         $ref: "#/CIDR"
     authorized_recipients:
+      description: |-
+        List of authorized fiscal codes that belong to the users
```
Comment (cloudify) [inline api/definitions.yaml]: I non empty, the service will be able to send messages only to these fiscal codes.

File: api/definitions.yaml [hunk 3]
```diff
@@ -368,13 +368,21 @@ Service:
...
+        this service can send messages to
       type: array
       items:
         $ref: "#/FiscalCode"
+    fiscal_code:
```
Comment (cloudify) [inline api/definitions.yaml]: `organization_fiscal_code`

File: lib/models/service.ts [hunk 4]
```diff
@@ -36,6 +36,8 @@ export const Service = t.interface({
   authorizedRecipients: readonlySetType(FiscalCode, "fiscal codes"),
   // the name of the department within the service
   departmentName: NonEmptyString,
+  // fiscal code of the service, used to receive payments
```
Comment (cloudify) [inline lib/models/service.ts]: this is the fiscal code of the organization, not the service

**Comment by gunzip** [comment]
renamed to organizationFiscalCode

**Pull request merged**